doc/user_guide/index.rst

```diff
@@ -38,32 +33,86 @@
    -----------------------------------------------------------------------------
    Written by R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
+.. only:: html
+
+    .. image:: ../logo/psyclone_v1.0.png
+        :width: 75%
+        :align: center
+        :alt: PSyclone
+
+PSyclone User's Guide
+=====================
+
+
+.. only:: html
+
+    PSyclone is a code generation system developed to support domain-specific
+    languages (DSLs) for finite element, finite volume and finite difference
+    codes. Notably, it is used in the
+    `LFRic Project <https://www.metoffice.gov.uk/research/modelling-systems/
+    lfric/>`_, and it also supports the `GOcean <https://puma.nerc.ac.uk/trac
+    /GOcean>`_ (2D, finite difference) DSL. In addition it is being extended
+    to support the processing of existing finite difference codes such as the 
+    `NEMO <https://www.nemo-ocean.eu/>`_ ocean model. 
+    Find more information in the
+    `Developer's Guide <https://psyclone-dev.readthedocs.io/>`_
+    and the
+    `Reference Guide <https://psyclone-ref.readthedocs.io/>`_.
+
+
 .. toctree::
-   :maxdepth: 3
-
-   introduction
-   getting_going
-   examples
-   psyclone_script
-   kernel_layer
-   algorithm_layer
-   psy_layer
-   psyir
-   transformations
-   distributed_memory
-   built_ins
-   dynamo0p3
-   gocean1p0
-   nemo
-   stub_gen
-   line_length
-   fortran_naming_conventions
-   api
-   profiling
-   psyke
-   system_specific_setup
-   configuration
-   zz_bibliography
+    :maxdepth: 2
+    :caption: Introduction
```

Comment:
As discussed, please could you alter this so that the PDF document has the same structure.

Reply:
Done